src/legacy/core_plugins/kibana/public/discover/np_ready/angular/directives/_index.scss

```diff
@@ -1,2 +1,3 @@
 @import 'no_results';
-@import 'histogram';
\ No newline at end of file
+@import 'histogram';
+@import './collapsible_sidebar/index';
```

Comment:
dear @elastic/kibana-design, no new SCSS inside, just change of path and imports

Reply:
I just reviewed the SCSS files that were moved to a new directory and LGTM!